libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerRecognizer.cs

```diff
@@ -300,5 +300,47 @@ protected virtual Task<IQnAMakerClient> GetQnAMakerClientAsync(DialogContext dc)
 
             return Task.FromResult<IQnAMakerClient>(new QnAMaker(endpoint, new QnAMakerOptions(), httpClient, TelemetryClient, logPersonalInfo));
         }
+
+        /// <summary>
+        /// Uses the RecognizerResult to create a list of properties to be included when tracking the result in telemetry.
+        /// </summary>
+        /// <param name="recognizerResult">Recognizer Result.</param>
+        /// <param name="telemetryProperties">A list of properties to append or override the properties created using the RecognizerResult.</param>
+        /// <param name="dialogContext">Dialog Context.</param>
+        /// <returns>A dictionary that can be included when calling the TrackEvent method on the TelemetryClient.</returns>
+        protected override Dictionary<string, string> FillRecognizerResultTelemetryProperties(RecognizerResult recognizerResult, Dictionary<string, string> telemetryProperties, DialogContext dialogContext = null)
```

Comment:
- Both of the `Botbuilder.AI` recognizers (`QnAMakerRecognizer & `LuisAdaptiveRecognizer`) do not have access to new base `AdaptiveRecognizer` class, unfortunately, as we are placing the base class in `Botbuilder.Dialogs.Adaptive`
- The `AI` recognizers will keep their own `LogPersonalInformation` property and `FillRecognizerResultTelemetryProperties` override (as opposed to pulling them from a base class), due to the dependency issue
- See https://github.com/microsoft/botbuilder-dotnet/pull/5066#issuecomment-775588252